tests/integration/aliases/step1/package.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "aliases",
+    "license": "Apache-2.0",
+    "main": "bin/index.js",
+    "typings": "bin/index.d.ts",
+    "devDependencies": {
+        "typescript": "^2.5.3"
```

Comment:
2.5.3?! :)

Reply:
Copied from other tests.  I'll leave to you to do an overall sweep and upgrade if you want :-).